app/assets/javascripts/items.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I prefer to remove empty files from the project to avoid bloat. You can always add them back later if you find out you need them.

Reply:
Okay, I'll do that. What about `.keep` files?

Reply:
You can `.keep` those.

![](https://media.giphy.com/media/3o7WIMPobHmxGPnYre/giphy.gif)